.github/workflows/test.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version: ${{ env.GO_VERSION }}
+          cache-dependency-path: src/github.com/containerd/nerdctl
```

Comment:
IMHO this is useless if we remove this line https://github.com/containerd/nerdctl/pull/2176/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R21

Reply:
I think we can add checkout depth : 1 for this job
```
with:
          fetch-depth: 1
```